### ISSUE TITLE
[Bug] The App suddenly crashes after updating to Android 12

### ISSUE THREAD

**Issue by JudeDerry** [Jays2Kings/tachiyomiJ2K#1115]
### Device information
* Tachiyomi version: Tachiyomi J2K v 1.4.3
* Android phone & version: Samsung S21 Ultra. Android 12.

## Steps to reproduce
1. Tap the App icon to open it. 

### Expected behavior
The App should open and start running after tapping the icon.

### Actual behavior
The App suddenly crashes.

### Other details
The App was working just fine before updating my phone to the latest Android version.

**Comment by Ruratae** [comment]
I'm also experiencing this, though mine started a little differently. I have several categories in the app including "Default" and some others. The app launched to my second category and I scrolled down. Once I reached some point in scrolling, possibly my third category, the app crashed. On subsequent launches the app scrolls the library down to that point and it crashes again.
Although I managed to make it to other screens, the app crashes when trying to view sources and browse from there. It also crashes when trying to read anything from Recents.
The settings menu still works.
Coincidentally, I'm on the exact same phone with the exact same version of Tachiyomi.
